applications/nrf_desktop/src/modules/dfu.c

```diff
@@ -829,6 +845,21 @@ static void handle_image_info_request(uint8_t *data, size_t *size)
 		uint8_t minor = 0;
 		uint16_t revision = 0;
 
+		if (is_semver_supported) {
+			LOG_INF("Booted application version: %d.%d.%d+%d",
```

Comment:
The `+%d` notation may be misleading as SUIT does not provide build number.
The notations is `%d.%d.%d-%s.%d` where `%s` is the pre-release name.

Reply:
improved this behaviour